lib/ansible/modules/service.py

```diff
@@ -80,7 +80,7 @@
     use:
         description:
         - The service module actually uses system specific modules, normally through auto detection, this setting can force a specific module.
-        - Normally it uses the value of the 'ansible_service_mgr' fact and falls back to the old 'service' module when none matching is found.
+        - Normally it uses the value of the C(ansible_service_mgr) fact and falls back to the old I(service) module when none matching is found.
```

Comment:
I'm not sure italic is the right way to emphasize this. The `service` action plugin falls back to `ansible.legacy.service` specifically.

Reply:
Oopsies, another one that slipped through the cracks, thanka for pointing it out! 😊